### PR TITLE
Upgrade `go-template-utils` to v3.2.0

### PR DESCRIPTION
**Pull request by dhaiducek**
ref: https://issues.redhat.com/browse/ACM-5032

### CODE REVIEW

**Review by gparvin — DISMISSED** [review]

File: build/common/config/.golangci.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ linters-settings:
     check-shadowing: false
   gci:
     sections:
-      - prefix(open-cluster-management.io/config-policy-controller)
+      - prefix(open-cluster-management.io/governance-policy-propagator)
```
Comment (JustinKuli) [inline build/common/config/.golangci.yml:83]: 😹

**Review by JustinKuli — APPROVED** [review]

File: build/common/config/.golangci.yml (hunk 1 above)
Comment (dhaiducek) [reply build/common/config/.golangci.yml:83]: Yeah--makes me wonder whether this happens elsewhere! 😢

File: Makefile [hunk 2]
```diff
@@ -87,15 +87,15 @@ clean:
...
+	$(call go-get-tool,github.com/daixiang0/gci@v0.10.1)
+	$(call go-get-tool,mvdan.cc/gofumpt@v0.5.0)
 
 # All available format: format-go format-protos format-python
 # Default value will run all formats, override these make target with your requirements:
 #    eg: fmt: format-go format-protos
 fmt: fmt-dependencies
 	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gofmt -s -w
-	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gci -w -local "$(shell cat go.mod | head -1 | cut -d " " -f 2)"
+	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gci write -s standard -s default -s "prefix($(shell cat go.mod | head -1 | cut -d " " -f 2))"
```
Comment (JustinKuli) [inline Makefile:98]: I thought I remembered there being a spot for this in a config file? But maybe that can't do the (somewhat spiffy) reading of the `go.mod`.

File: build/common/config/.golangci.yml (hunk 1 above)
Comment (JustinKuli) [reply build/common/config/.golangci.yml:79]: This is probably the config file I was thinking of... I need more coffee 😂

**Review by JustinKuli — APPROVED** [review]

**Pull request merged**